app/controllers/billings_controller.rb

```diff
@@ -0,0 +1,39 @@
+class BillingsController < ApplicationController
```

Comment:
From @lacuchilla:
I'm seeing a lot of empty methods in this controller. Were they put in only to render views?